app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,33 @@
+class GroupsController < ApplicationController
+  def index
+    @groups = Group.all
+  end
+
+  def new
+    @group = Group.new
+  end
+
+  def create
+    @group = Group.create(group_params)
+
+    respond_with @group, location: -> { groups_path }
+  end
+
+  private
+
+  def group_params
+    params.require(:group).permit(
+      :name,
+      :responsable_name,
+      :responsable_contact,
+      :from_date,
+      :to_date,
+      :notes,
+    )
+  end
+
+  def group
+    @group ||= Group.find(params[:id])
+  end
+  helper_method :group
```

Comment:
Is this method used at the moment? If not, it should be removed from this PR :)

Reply:
Actually no...I was looking ahead! :)
Removed with fixup in `0212a18`